### ISSUE TITLE
didn't work when the sampling points were too dense

### ISSUE THREAD

**Issue by fgmn** [marmotlab/large-scale-DRL-exploration#4]
We found that the algorithm didn't work when the sampling points were too dense i.e. set NODE_RESOLUTION = 1.0 originally 4.0. This is necessary in some scenes with narrow passages

**Comment by caoyuhong001** [comment]
That’s true. In this case you may need to apply graph rarefaction so the planner can keep the neighboring nodes dense and will not get stuck in large scale environments. You can check our ROS planner repo. When I did my real robot experiment I set the node resolution to 1m or 0.8m as well, but for sure higher resolution will influence the scalability.
